docs/installation.rst

```diff
@@ -132,6 +151,24 @@ By default, Robusta sends notifications when Kubernetes pods crash.
 
    kubectl delete deployment crashpod
 
+Installing a second cluster
+---------------------------------
+
+When installing a second cluster on the same account, there's no need to run ``robusta gen-config`` again.
+Just follow the :ref:`Standard Installation` instruction using the ``generated_values.yaml`` file you've already created.
```

Comment:
I would explicitly give the command here.

When installing a second cluster on the same account, there's no need to run ``robusta gen-config`` again.
Using your existing ``generated_values.yaml`` and the new `clusterName` run:

helm install robusta robusta/robusta -f ./generated_values.yaml --set clusterName=<YOUR_NEW_CLUSTER_NAME>